ruby_event_store/lib/ruby_event_store/spec/event_repository_lint.rb

```diff
@@ -106,6 +222,25 @@
     ], 'stream', 1)
   end
 
+  specify ':auto queries for last position in given stream when linking' do
+    skip unless test_expected_version_auto
+    skip unless test_link_events_to_stream
+    repository.append_to_stream([
+      eventA = TestDomainEvent.new(event_id: SecureRandom.uuid),
+      eventB = TestDomainEvent.new(event_id: SecureRandom.uuid),
+      eventC = TestDomainEvent.new(event_id: SecureRandom.uuid),
+    ], 'another', :auto)
+    repository.append_to_stream([
+      event0 = TestDomainEvent.new(event_id: SecureRandom.uuid),
+      event1 = TestDomainEvent.new(event_id: SecureRandom.uuid),
+    ], 'stream', :auto)
+    repository.link_to_stream([
+      eventA.event_id,
+      eventB.event_id,
+      eventC.event_id,
+    ], 'stream', 1)
```

Comment:
what's the expectation here?

Reply:
Inspired by `specify ':auto queries for last position in given stream' do` probably that you can mix `:auto` and `Integer` in that order. Although I can't say the name reveals that...

Reply:
The expectation is no exception.

Reply:
Making a query would probably be a better idea.

Reply:
expect{ ... }.not_to raise_error would be a clear indication of that